pkg/apis/serving/v1alpha1/framework_scikit.go

```diff
@@ -43,7 +48,7 @@ func (s *SKLearnSpec) CreateModelServingContainer(modelName string, config *Fram
 		Resources: s.Resources,
 		Args: []string{
 			"--model_name=" + modelName,
-			"--model_dir=" + s.ModelURI,
+			"--model_dir=" + constants.DefaultModelLocalMountPath,
```

Comment:
I am wondering how this would work with knative revisions, if user only canary model uri but here we are mutating to a constant model path, does this mean knative would never create a new revision ?

Reply:
the annotation is different, that causes a new revision to be created...
```
$ kubectl get revision
NAME                                   SERVICE NAME                           GENERATION   READY   REASON
tensorrt-simple-string-default-6c6p9   tensorrt-simple-string-default-6c6p9   2            True    
tensorrt-simple-string-default-b5w49   tensorrt-simple-string-default-b5w49   1            True    
```

Reply:
Ah, can we leave some comments for that annotation ? It is quite difficult for people to figure out what's going on there.